angr/engines/light/engine.py

```diff
@@ -53,17 +53,20 @@ def _codeloc(self, block_only=False):
 
 class SimEngineLightVEXMixin:
 
-    def _process(self, state, successors, block=None, whitelist=None):  # pylint:disable=arguments-differ,unused-argument
+    def _process(self, state, successors, block=None, whitelist=None, *args, **kwargs):  # pylint:disable=arguments-differ,unused-argument
```

Comment:
Can you do `*args` after keyword arguments in Python 3.5?

Python 3.5 is the earliest version of Python that we are still supporting.

Reply:
If I interpret the grammar correctly, yes: https://docs.python.org/3.5/reference/expressions.html#calls .

Reply:
If I read this grammar properly, it seems you can: https://docs.python.org/3.5/reference/expressions.html#calls .

Reply:
Oh I didn't know it was added since Python 3.5.